qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -98,12 +105,12 @@ class DefaultPoliciesTest extends BaseSpecification {
             .addLabel("app", "test")
             .setCommand(["sleep", "600"]),
         new Deployment()
-            .setName(NGINX_1_10)
-            .setImage("quay.io/rhacs-eng/qa:docker-io-nginx-1-10")
+            .setName(TRIGGER_MOST)
+            .setImage("quay.io/rhacs-eng/qa-multi-arch:trigger-policy-violations-most")
             .addLabel("app", "test"),
         new Deployment()
             .setName(GCR_NGINX)
-            .setImage("us.gcr.io/stackrox-ci/nginx:1.11.1")
+            .setImage("quay.io/rhacs-eng/qa-multi-arch:nginx-1.12")
```

Comment:
This is changing the test behavior and is inconsistent -- the name is `GCR_NGINX` but image is on quay. I think we should push the image to gcr.io.

Reply:
Even though the deployment name is `GCR_NGINX`, looking at the test functionality, the deployment is only used to verify `Fixable CVSS >= 7` policy violations. So I guess any image should be fine. If needed we can correct the deployment name, Let me know.

Reply:
I've pushed the multi-arch image to gcr. (us.gcr.io/stackrox-ci/qa-multi-arch:nginx-1.12)